rickchurch/constants.py

```diff
@@ -3,18 +3,24 @@
 from decouple import config
 
 
+DISCORD_BASE_URL = "https://discord.com/api"
+
 log_level: str = config("LOG_LEVEL", default="INFO")
 
 base_url: str = config("BASE_URL")  # URL to host church of rick
-discord_token_url: str = config("TOKEN_URL", default="https://discord.com/api/oauth2/token")
-discord_user_url: str = config("USER_URL", default="https://discord.com/api/users/@me")
 
 # Get these from https://discord.com/developers/applications, OAuth2 section
 client_id: str = config("CLIENT_ID")
 client_secret: str = config("CLIENT_SECRET")
 # Get this by adding {base_url}/oauth_callback as URI in the application's OAuth2 section
 oauth_redirect_url: str = config("OAUTH_REDIRECT_URL")
 
+enable_auto_join: bool = bool(config("ENABLE_DISCORD_AUTOJOIN", ""))
+discord_guild_id = discord_bot_token = ""
+if enable_auto_join:
+    discord_guild_id = config("DISCORD_GUILD_ID")
+    discord_bot_token = config("DISCORD_BOT_TOKEN")
```

Comment:
Type-hint these as strings, pyright might be causing problems to functions that expect string and get this return type from `decouple.config`, since it returns a union of many different types.

Reply:
They are already declared as strings 2 lines above

Reply:
Oh, in that case, just use a teranry conditional for both, I think it would be cleaner